packages/web/modals/validator-squad.tsx

```diff
@@ -110,10 +120,45 @@ const ValidatorSquadContent: FunctionComponent<ValidatorSquadContentProps> =
         {
           id: "validatorSquadTable",
           columns: [
-            {
-              accessorKey: "validatorName",
+            columnHelper.accessor((row) => row, {
+              cell: observer((props: CellContext<any, any>) => {
+                return (
+                  <div className="flex items-center gap-3">
+                    {/*  input placeholder */}
+                    <input type="radio" />
+                    <div className="h-10 w-10 overflow-hidden rounded-full">
+                      <img
+                        alt={props.row.original.validatorName}
+                        src={props.row.original.imageUrl || ""}
+                      />
+                    </div>
+                    <div className="flex flex-col">
+                      <div className="subtitle1 md:subtitle2">
+                        {props.row.original.validatorName}
+                      </div>
+                      {!!props.row.original.website && (
```

Comment:
@JoseRFelix has been bringing it up and I agree:

IMO `Boolean()` constructor is easier to read/reason about over `!!`